python/whylogs/api/writer/local.py

```diff
@@ -15,16 +20,26 @@ def __init__(self, base_dir: Optional[str] = None, base_name: Optional[str] = No
         self._base_dir = base_dir
         self._base_name = base_name
 
-    def write(self, profile: DatasetProfileView, dest: Optional[str] = None) -> None:
-        if dest is None:
+    def write(
+        self,
+        file: Optional[Writable] = None,
+        profile: Optional[DatasetProfileView] = None,
+        dest: Optional[str] = None,
+        **kwargs,
+    ) -> None:
+        if profile:
+            logger.warning("`profile will be deprecated, you should use `file` instead")
```

Comment:
I think there are packages that supply decorators to deprecate function arguments.